src/components/userProfile/UserProfile.js

```diff
@@ -52,6 +53,18 @@ class UserProfile extends Component {
       let currentPosition = this.state.currentPosition;
       let currentLocation = this.state.currentLocation;
 
+      let sectionIDs = ['Experience', 'Education'];
+      let i;
+      let j;
```

Comment:
It was used at previous code which is deleted and I forgot removing these. Thanks!